README.md

```diff
@@ -84,3 +84,7 @@ this issue, clean all the compiled files from the source code:
 ```bash
 make distclean
 ```
+
+## Packages
```

Comment:
```suggestion
## Available Linux packages
```

... just to avoid confusion with Win/Mac/etc.

Reply:
Mhh, just discovering "MacPorts" in the list :-)

Reply:
Yes your are right. Better section name will be 'Available GNU/Linux and Unix-like OS packages', because list contains FreeBSD and macOS.